testdata/testing/configured/defaults/config.json

```diff
@@ -1,19 +1,20 @@
 {
-  "mqtt_server_uri":"tcp://localhost:11883",
-  "mqtt_client_id_prefix":"easee",
+  "mqtt_server_uri": "tcp://localhost:11883",
+  "mqtt_client_id_prefix": "easee",
   "work_dir": "../testdata/testing/configured",
   "log_level": "debug",
   "log_format": "text",
   "configured_at": "9998-12-31T00:00:00+01:00",
   "accessToken": "test.access.token",
   "refreshToken": "test.refresh.token",
   "expiresAt": "9999-01-01T00:00:00Z",
-  "easeeBaseURL": "http://localhost:9999",
+  "easeeBaseURL": "http://localhost:9998",
```

Comment:
Do you need that field at all?

Reply:
I'm gonna leave it as I'd like to extend the test suite in the future